controller-server/src/main/java/com/yahoo/vespa/hosted/controller/application/DeploymentJobs.java

```diff
@@ -75,21 +76,21 @@ public DeploymentJobs withTriggering(JobType jobType,
                                       change.isPresent() && change.get() instanceof Change.VersionChange,
                                       triggerTime);
         });
-        return new DeploymentJobs(projectId, status, jiraIssueId);
+        return new DeploymentJobs(projectId, status, issueId);
     }
 
     public DeploymentJobs withProjectId(long projectId) {
-        return new DeploymentJobs(Optional.of(projectId), status, jiraIssueId);
+        return new DeploymentJobs(Optional.of(projectId), status, issueId);
     }
 
-    public DeploymentJobs withJiraIssueId(Optional<String> jiraIssueId) {
-        return new DeploymentJobs(projectId, status, jiraIssueId);
+    public DeploymentJobs withIssueId(IssueId issueId) {
```

Comment:
Now that it's a proper type, why not `with(IssueId issueId)`?

Reply:
Yes, will change that.

Reply:
Done.